api/Controllers/WorkflowsController.cs

```diff
@@ -110,7 +110,7 @@ [FromBody] AnonymizerDoneNotification notification
     [Route("notify-constant-level-oiler-done")]
     [ProducesResponseType(StatusCodes.Status200OK)]
     [ProducesResponseType(StatusCodes.Status404NotFound)]
-    public ActionResult<PlantDataResponse> ConstantLevelOilerDone(
+    public async Task<ActionResult<PlantDataResponse>> ConstantLevelOilerDone(
```

Comment:
```suggestion
    public async Task<ActionResult<PlantDataResponse>> ConstantLevelOilerCompleted(
```

I understand this is scope creep so no need to fix it in this PR